contracts/axelar-gas-service/src/test.rs

```diff
@@ -22,14 +26,87 @@ fn setup_env<'a>() -> (Env, Address, Address, AxelarGasServiceClient<'a>) {
 
     let client = AxelarGasServiceClient::new(&env, &contract_id);
     let gas_collector: Address = Address::generate(&env);
-    client.initialize(&gas_collector);
 
     (env, contract_id, gas_collector, client)
 }
 
+#[test]
+fn test_initialize() {
+    let (env, contract_id, gas_collector, client) = setup_env();
+    client.initialize(&gas_collector);
+
+    assert!(env.as_contract(&contract_id, || {
+        env.storage()
+            .instance()
+            .get::<DataKey, bool>(&DataKey::Initialized)
+            .unwrap()
```

Comment:
use assert_ok instead of unwrap this isn't just helper code but what we're testing for

Reply:
my thinking was that it's not enough to just test for `Ok(_)` but also that the `DataKey::Initialized` is set to `true`, hence the `unwrap` to pull the value out of the `Option`. could maybe do something like this to avoid the unwrap:
```rs
    assert_eq!(
        env.as_contract(&contract_id, || {
            env.storage()
                .instance()
                .get::<DataKey, bool>(&DataKey::Initialized)
        }),
        Some(true)
    );
```

but it's slightly janky. Would you want to just test for existence instead of ensuring the values match?

Reply:
Add an `assert_some!` macro that similar to `assert_ok` tests that it's a `Some(value)` and returns the unwrapped value, so we can still test if the output is `true`